apps/builder/app/shared/router-utils/path-utils.ts

```diff
@@ -55,6 +56,38 @@ export const builderUrl = (props: {
   return url.href;
 };
 
+export const builderUrlNew = (props: {
```

Comment:
I would rather call this builderUrl and rename the other one to builderUrlDeprecated so that its absolutely clear what to use

Reply:
or is the plan to replace the old one entirely in the next step?

Reply:
Plan to replace entirely